exercicios/para-casa/entregas/exemplo-nome-sobrenome/contaBancaria.test.js

```diff
@@ -0,0 +1,52 @@
+const ContaBancaria = require("./ContaBancaria");
+
+describe("Testar transação de saque", () => {
+  test("Sacar sem limite e saldo", () => {
+    const pessoaBancaria = new ContaBancaria(0, 0);
+    const isValorSacado = pessoaBancaria.sacarDinheiro(600);
+
+    expect(isValorSacado).toBe(false);
+  });
+
+  test("Sacar sem limite e com saldo", () => {
+    const pessoaBancaria = new ContaBancaria(350, 0);
+    const isValorSacado = pessoaBancaria.sacarDinheiro(350);
+
+    expect(isValorSacado).toBe(true);
+    expect(pessoaBancaria.getSaldoDaConta()).toBe(0);
+  });
+  test("Sacar com limite e sem saldo", () => {
+    const pessoaBancaria = new ContaBancaria(0, 150);
+    const isValorSacado = pessoaBancaria.sacarDinheiro(150);
+
+    expect(isValorSacado).toBe(true);
+    expect(pessoaBancaria.getSaldoDaConta()).toBe(-150);
```

Comment:
Dica: A prof falou pra a gente usar toEqual quando for numeros, eu acredito que seja boa pratica também, mas fique a vontade para tirar essa duvida com ela  : )